inc/osvr/Client/ViewerEye.h

```diff
@@ -109,15 +110,23 @@ namespace client {
         getProjection(double near, double far,
                       OSVR_MatrixConventions flags) const;
 
+        /// @brief Gets clipping planes for a given surface
+        OSVR_CLIENT_EXPORT util::Rectd getRect() const;
+
         Viewport getDisplayRelativeViewport() const { return m_viewport; }
 
+        OSVR_DisplayInputCount getDisplayInputIdx() const {
+            return m_displayInputIdx;
+        }
+
       private:
         friend class DisplayConfigFactory;
         ViewerEye(
             OSVR_ClientContext ctx, Eigen::Vector3d const &offset,
             const char path[], Viewport &&viewport, util::Rectd &&unitBounds,
             bool rot180, double pitchTilt,
-            boost::optional<OSVR_RadialDistortionParameters> radDistortParams);
+            boost::optional<OSVR_RadialDistortionParameters> radDistortParams,
+            OSVR_DisplayInputCount displayInputIdx);
```

Comment:
sorry I will have caused a merge conflict here. I'll do the merge myself so you don't have to rebase.